packages/vehicles/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@opentripplanner/vehicles",
```

Comment:
Since this package exports an overlay that is to be used with the `@opentripplanner/base-map` component, perhaps this package should be renamed to `vehicle-overlay`. Also, there's already a `vehicle-rental-overlay` package, so to further distinguish this package, perhaps it could be called `transit-vehicle-overlay`.

Reply:
sounds good...working on that...

Reply:
fixed